sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/Samples/Sample01_DataFeedCrudOperations.cs

```diff
@@ -143,13 +143,13 @@ public async Task UpdateDataFeedAsync()
             string originalDescription = dataFeed.Description;
             dataFeed.Description = "This description was generated by a sample.";
 
-            await adminClient.UpdateDataFeedAsync(dataFeedId, dataFeed);
```

Comment:
consider same changes for samples here as the other PR

Reply:
Removed extra GET and printed updated description.